qiskit_nature/constants.py

```diff
@@ -38,3 +38,19 @@
 KB_J_PER_K = 1.3806488e-23
 BOHR = 0.52917721092  # No of Angstroms in Bohr (from 2010 CODATA)
 DEBYE = 0.393430307  # No ea0 in Debye. Use to convert our dipole moment numbers to Debye
+
+PERIODIC_TABLE = [
```

Comment:
Nice, you fixed it! I noticed the other week that black had altered it and thought it was much better in the table form as it was created.

Reply:
I actually extracted this from the `FreezeCoreTransformer`. I didn't know that this was duplicated in the `QMolecule`. Same as some other constants. Since the `QMolecule` is no loinger standalone since we added the deprecation notice to it anyways, I just refactored that too :+1:

Reply:
I imagine the copy in FreezeTransformer originated from QMolecule - its a better place in constants of course!